src/opnsense/mvc/app/models/OPNsense/Dnsmasq/Dnsmasq.xml

```diff
@@ -47,7 +47,10 @@
             <no_interface type="InterfaceField">
                 <Multiple>Y</Multiple>
             </no_interface>
-            <fqdn type="BooleanField"/>
+            <fqdn type="BooleanField">
+                <Required>Y</Required>
+                <Default>1</Default>
```

Comment:
@AdSchellevis  @Monviech  Any apply without an explicit user toggle on the option will have set this value to "0". Should we flip this with a migration?